layouts/topics/kubernetes.html

```diff
@@ -1,9 +1,35 @@
 {{ define "hero" }}
     <header class="header-hero header-hero-glow">
         <div class="header-hero-items container mx-auto">
-            <div class="header-hero-item">
+            <div class="header-hero-item pr-20">
                 <h1>{{ .Params.hero.title }}</h1>
                 {{ .Params.hero.body | markdownify }}
+                <h3 class="text-white">Get Started Today</h3>
+                <div>
+                    <div class="flex flex-row max-w-full">
+                        <a class="btn bg-gray-300 border border-gray-300 hover:bg-gray-200 mb-5 flex justify-center w-1/2 mr-4"
+                            href="https://app.pulumi.com/signup?reason=pass_through&type=github.com">
+                            <img class="h-10 my-auto mr-4" src="/logos/tech/github.svg" alt="GitHub">
+                            <p class="hero--registration-button my-auto">GitHub</p>
+                        </a>
+                        <a class="btn bg-gray-300 border border-gray-300 hover:bg-gray-200 mb-5 flex justify-center w-1/2 mr-4"
+                            href="https://app.pulumi.com/signup?reason=pass_through&type=gitlab.com">
+                            <img class="h-10 my-auto mr-4" src="/logos/tech/gitlab.svg" alt="GitLab">
+                            <p class="hero--registration-button my-auto">GitLab</p>
+                        </a>
+                    </div>
+                    <div class="flex flex-row max-w-full">
+                        <a class="btn bg-gray-300 border border-gray-300 hover:bg-gray-200 mb-5 flex justify-center w-1/2 mr-4"
+                            href="https://app.pulumi.com/signup?reason=pass_through&type=bitbucket.com">
+                            <img class="h-10 my-auto mr-4" src="/logos/tech/atlassian.svg" alt="BitBucket">
+                            <p class="hero--registration-button my-auto">BitBucket</p>
```

Comment:
The official Bitbucket brand name is spelled with a lower-case 'b' for 'bucket' unlike GitHub and GitLab.

```suggestion
                            <p class="hero--registration-button my-auto">Bitbucket</p>
```